lms/templates/edraak_university/instructor/list.html

```diff
@@ -29,7 +29,7 @@ <h1>${_('Edraak University IDs')}</h1>
                         </tr>
                     </thead>
                     % for uni_id in object_list:
-                        <tr class="university-id-entry" class="${'conflicted' if uni_id.is_conflicted else ''}">
+                        <tr class="${'university-id-entry conflicted' if uni_id.is_conflicted else 'university-id-entry'}">
```

Comment:
```suggestion
                        <tr class="university-id-entry ${'conflicted' if uni_id.is_conflicted else ''}">
```

This is more succinct.